bookkeeper-benchmark/src/main/java/org/apache/bookkeeper/benchmark/BenchReadThroughputLatency.java

```diff
@@ -102,14 +102,14 @@ private static void readLedger(ClientConfiguration conf, long ledgerId, byte[] p
                 }
                 long starttime = System.nanoTime();
 
-                while (entriesRead <= lastConfirmed) {
+                while (entriesRead < lastConfirmed) {
                     long nextLimit = lastRead + 100000;
                     Enumeration<LedgerEntry> entries;
                     if (batchEntries <= 0) {
                         long readTo = Math.min(nextLimit, lastConfirmed);
                         entries = lh.readEntries(lastRead + 1, readTo);
                     } else {
-                        entries = lh.batchReadEntries(lastRead, batchEntries, -1);
+                        entries = lh.batchReadEntries(lastRead + 1, batchEntries, -1);
```

Comment:
The `lastRead` entry already be read, so we should read next entry at the next read.

Reply:
Do we need init the `lastRead` to `-1` instead of `0`, otherwise the entry `0` can't be read

Reply:
Nice catch.